CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to Climate Connect
+
+Thank you for your interest in contributing to ClimateConnect!
+
+Table of Contents:
+
+1. [Feature Requests](#feature-requests)
+1. [Bug Reports](#bug-reports)
+1. [Pull Requests](#pull-requests)
+    1. [Documentation](#documentation)
+1. [Contact](#contact)
+
+## Feature Requests
+Feature requests should be reported in the [Climate Connect issue tracker](https://github.com/climateconnect/climateconnect/issues). To reduce the number of duplicates, please make sure to check existing issues.
```

Comment:
```suggestion
Feature requests should be reported in the [Climate Connect issue tracker](https://github.com/climateconnect/climateconnect/issues). To avoid duplicates, please make sure to check existing issues.
```